app/src/main/java/net/squanchy/injection/ContextModule.java

```diff
@@ -0,0 +1,21 @@
+package net.squanchy.injection;
+
+import android.content.Context;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module
+public class ContextModule {
```

Comment:
Hmm no the point for the activity context module is to provide an Activity context. It will break a lot of things if you make it return anything that is not the current activity. Unfortunately contexts are not interchangeable, even though they are all the same type (the reason it provides a context instead of activity is simply because we don't need to use it as activity anywhere, but we must be sure it is an activity context, hence the fact it ingests activity). If you need to use an application context you can still use `getApplicationContext()` out of that. If you want to provide the application context somewhere, let the Application component provide an Application and use that instead.